### ISSUE TITLE
`Unknown column 'commerce_orders.orderSiteId' in 'field list'` right after installation

### ISSUE THREAD

**Issue by Saboteur777** [craftcms/commerce#1829]
**Description**
`/control-panel/commerce/orders` throws this error on a new installation: 

```
PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'commerce_orders.orderSiteId' in 'field list' in /var/www/vendor/yiisoft/yii2/db/Command.php:1293
Stack trace:
#0 /var/www/vendor/yiisoft/yii2/db/Command.php(1293): PDOStatement->execute()
#1 /var/www/vendor/yiisoft/yii2/db/Command.php(1159): yii\db\Command->internalExecute()
#2 /var/www/vendor/yiisoft/yii2/db/Command.php(427): yii\db\Command->queryInternal()
#3 /var/www/vendor/yiisoft/yii2/db/Query.php(444): yii\db\Command->queryScalar()
#4 /var/www/vendor/craftcms/cms/src/db/Query.php(214): yii\db\Query->exists()
#5 /var/www/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1539): craft\db\Query->exists()
#6 /var/www/vendor/twig/twig/src/Extension/CoreExtension.php(1499): craft\elements\db\ElementQuery->exists()
#7 /var/www/vendor/craftcms/cms/src/helpers/Template.php(92): twig_get_attribute()
#8 /var/www/storage/runtime/compiled_templates/00/008da45e27405ae6b9c4e2dff9c00f6003fb19ed58af0ce87980b3b783f0549d.php(128): craft\helpers\Template::attribute()
#9 /var/www/vendor/twig/twig/src/Template.php(405): __TwigTemplate_b752695f39c8f0c3dbec0fa42404cbf9aa756e00ed077202b314f2c61534e8ea->doDisplay()
#10 /var/www/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#11 /var/www/storage/runtime/compiled_templates/8d/8d6e70d641191e80b41d14742e34ea7dc44a91bedd27f4d778f3bb0fc99cf91e.php(134): Twig\Template->display()
#12 /var/www/vendor/twig/twig/src/Template.php(182): __TwigTemplate_592a0ad893fa74730ff9aa57ba13203120e485c7f40bdae8cdc45224f6e97d61->block_toolbar()
#13 /var/www/vendor/twig/twig/src/Template.php(254): Twig\Template->displayBlock()
#14 /var/www/storage/runtime/compiled_templates/43/43513d41673ad87f564363ddfb307ae6eb35ae0a2a6e110e6e4538b6abc95920.php(97): Twig\Template->renderBlock()
#15 /var/www/vendor/twig/twig/src/Template.php(405): __TwigTemplate_0ab46c0a16323ce42f220e35324b6fffe4c0f04bb34d4c498093f508ea72afe1->doDisplay()
#16 /var/www/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#17 /var/www/storage/runtime/compiled_templates/8d/8d6e70d641191e80b41d14742e34ea7dc44a91bedd27f4d778f3bb0fc99cf91e.php(71): Twig\Template->display()
#18 /var/www/vendor/twig/twig/src/Template.php(405): __TwigTemplate_592a0ad893fa74730ff9aa57ba13203120e485c7f40bdae8cdc45224f6e97d61->doDisplay()
#19 /var/www/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#20 /var/www/storage/runtime/compiled_templates/8f/8f7402d53ba59939f51da872ef5a32561a8d9fa0abbcfe518474f8c0636388bb.php(63): Twig\Template->display()
#21 /var/www/vendor/twig/twig/src/Template.php(405): __TwigTemplate_856c9a7468d2047aa83872d3500f724f2fb903bbe1205040b0a2ae5219d21474->doDisplay()
#22 /var/www/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#23 /var/www/vendor/twig/twig/src/Template.php(390): Twig\Template->display()
#24 /var/www/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render()
#25 /var/www/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render()
#26 /var/www/vendor/craftcms/cms/src/web/View.php(392): Twig\Environment->render()
#27 /var/www/vendor/craftcms/cms/src/web/View.php(453): craft\web\View->renderTemplate()
#28 /var/www/vendor/craftcms/cms/src/web/Controller.php(238): craft\web\View->renderPageTemplate()
#29 /var/www/vendor/craftcms/commerce/src/controllers/OrdersController.php(96): craft\web\Controller->renderTemplate()
#30 [internal function]: craft\commerce\controllers\OrdersController->actionOrderIndex()
#31 /var/www/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#32 /var/www/vendor/yiisoft/yii2/base/Controller.php(180): yii\base\InlineAction->runWithParams()
#33 /var/www/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction()
#34 /var/www/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction()
#35 /var/www/vendor/craftcms/cms/src/web/Application.php(274): yii\base\Module->runAction()
#36 /var/www/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction()
#37 /var/www/vendor/craftcms/cms/src/web/Application.php(259): yii\web\Application->handleRequest()
#38 /var/www/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest()
#39 /var/www/web/index.php(21): yii\base\Application->run()
#40 {main}

Next yii\db\Exception: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'commerce_orders.orderSiteId' in 'field list'
The SQL being executed was: SELECT EXISTS(SELECT `elements`.`id`, `elements`.`fieldLayoutId`, `elements`.`uid`, `elements`.`enabled`, `elements`.`archived`, `elements`.`dateCreated`, `elements`.`dateUpdated`, `elements_sites`.`id` AS `siteSettingsId`, `elements_sites`.`slug`, `elements_sites`.`siteId`, `elements_sites`.`uri`, `elements_sites`.`enabled` AS `enabledForSite`, `commerce_orders`.`id`, `commerce_orders`.`number`, `commerce_orders`.`reference`, `commerce_orders`.`couponCode`, `commerce_orders`.`orderStatusId`, `commerce_orders`.`dateOrdered`, `commerce_orders`.`email`, `commerce_orders`.`isCompleted`, `commerce_orders`.`datePaid`, `commerce_orders`.`currency`, `commerce_orders`.`paymentCurrency`, `commerce_orders`.`lastIp`, `commerce_orders`.`orderLanguage`, `commerce_orders`.`message`, `commerce_orders`.`returnUrl`, `commerce_orders`.`cancelUrl`, `commerce_orders`.`billingAddressId`, `commerce_orders`.`shippingAddressId`, `commerce_orders`.`estimatedBillingAddressId`, `commerce_orders`.`estimatedShippingAddressId`, `commerce_orders`.`shippingMethodHandle`, `commerce_orders`.`gatewayId`, `commerce_orders`.`paymentSourceId`, `commerce_orders`.`customerId`, `commerce_orders`.`dateUpdated`, `commerce_orders`.`registerUserOnOrderComplete`, `commerce_orders`.`recalculationMode`, `commerce_orders`.`origin`, `commerce_orders`.`dateAuthorized`, `commerce_orders`.`totalPrice` AS `storedTotalPrice`, `commerce_orders`.`totalPaid` AS `storedTotalPaid`, `commerce_orders`.`itemTotal` AS `storedItemTotal`, `commerce_orders`.`totalDiscount` AS `storedTotalDiscount`, `commerce_orders`.`totalShippingCost` AS `storedTotalShippingCost`, `commerce_orders`.`totalTax` AS `storedTotalTax`, `commerce_orders`.`totalTaxIncluded` AS `storedTotalTaxIncluded`, `commerce_orders`.`shippingMethodName`, `commerce_orders`.`itemSubtotal` AS `storedItemSubtotal`, `commerce_orders`.`orderSiteId`, `content`.`id` AS `contentId`, `elements`.`draftId`, `drafts`.`sourceId` AS `draftSourceId`, `drafts`.`creatorId` AS `draftCreatorId`, `drafts`.`name` AS `draftName`, `drafts`.`notes` AS `draftNotes`, `drafts`.`trackChanges` AS `draftTrackChanges`, `drafts`.`dateLastMerged` AS `draftDateLastMerged`
FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `elementsSitesId`, `content`.`id` AS `contentId`
FROM `elements` `elements`
INNER JOIN `commerce_orders` `commerce_orders` ON `commerce_orders`.`id` = `elements`.`id`
LEFT JOIN `commerce_addresses` `billing_address` ON `billing_address`.`id` = `commerce_orders`.`billingAddressId`
LEFT JOIN `commerce_addresses` `shipping_address` ON `shipping_address`.`id` = `commerce_orders`.`shippingAddressId`
INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id`
INNER JOIN `content` `content` ON `content`.`elementId` = `elements`.`id`
INNER JOIN `drafts` `drafts` ON `drafts`.`id` = `elements`.`draftId`
WHERE (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`drafts`.`sourceId` IS NULL) AND (`elements`.`revisionId` IS NULL)
ORDER BY `commerce_orders`.`id`
LIMIT 1) `subquery`
INNER JOIN `commerce_orders` `commerce_orders` ON `commerce_orders`.`id` = `subquery`.`elementsId`
LEFT JOIN `commerce_addresses` `billing_address` ON `billing_address`.`id` = `commerce_orders`.`billingAddressId`
LEFT JOIN `commerce_addresses` `shipping_address` ON `shipping_address`.`id` = `commerce_orders`.`shippingAddressId`
INNER JOIN `elements` `elements` ON `elements`.`id` = `subquery`.`elementsId`
INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`elementsSitesId`
INNER JOIN `content` `content` ON `content`.`id` = `subquery`.`contentId`
INNER JOIN `drafts` `drafts` ON `drafts`.`id` = `elements`.`draftId`
ORDER BY `commerce_orders`.`id`) in /var/www/vendor/yiisoft/yii2/db/Schema.php:677
Stack trace:
#0 /var/www/vendor/yiisoft/yii2/db/Command.php(1298): yii\db\Schema->convertException()
#1 /var/www/vendor/yiisoft/yii2/db/Command.php(1159): yii\db\Command->internalExecute()
#2 /var/www/vendor/yiisoft/yii2/db/Command.php(427): yii\db\Command->queryInternal()
#3 /var/www/vendor/yiisoft/yii2/db/Query.php(444): yii\db\Command->queryScalar()
#4 /var/www/vendor/craftcms/cms/src/db/Query.php(214): yii\db\Query->exists()
#5 /var/www/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1539): craft\db\Query->exists()
#6 /var/www/vendor/twig/twig/src/Extension/CoreExtension.php(1499): craft\elements\db\ElementQuery->exists()
#7 /var/www/vendor/craftcms/cms/src/helpers/Template.php(92): twig_get_attribute()
#8 /var/www/storage/runtime/compiled_templates/00/008da45e27405ae6b9c4e2dff9c00f6003fb19ed58af0ce87980b3b783f0549d.php(128): craft\helpers\Template::attribute()
#9 /var/www/vendor/twig/twig/src/Template.php(405): __TwigTemplate_b752695f39c8f0c3dbec0fa42404cbf9aa756e00ed077202b314f2c61534e8ea->doDisplay()
#10 /var/www/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#11 /var/www/storage/runtime/compiled_templates/8d/8d6e70d641191e80b41d14742e34ea7dc44a91bedd27f4d778f3bb0fc99cf91e.php(134): Twig\Template->display()
#12 /var/www/vendor/twig/twig/src/Template.php(182): __TwigTemplate_592a0ad893fa74730ff9aa57ba13203120e485c7f40bdae8cdc45224f6e97d61->block_toolbar()
#13 /var/www/vendor/twig/twig/src/Template.php(254): Twig\Template->displayBlock()
#14 /var/www/storage/runtime/compiled_templates/43/43513d41673ad87f564363ddfb307ae6eb35ae0a2a6e110e6e4538b6abc95920.php(97): Twig\Template->renderBlock()
#15 /var/www/vendor/twig/twig/src/Template.php(405): __TwigTemplate_0ab46c0a16323ce42f220e35324b6fffe4c0f04bb34d4c498093f508ea72afe1->doDisplay()
#16 /var/www/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#17 /var/www/storage/runtime/compiled_templates/8d/8d6e70d641191e80b41d14742e34ea7dc44a91bedd27f4d778f3bb0fc99cf91e.php(71): Twig\Template->display()
#18 /var/www/vendor/twig/twig/src/Template.php(405): __TwigTemplate_592a0ad893fa74730ff9aa57ba13203120e485c7f40bdae8cdc45224f6e97d61->doDisplay()
#19 /var/www/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#20 /var/www/storage/runtime/compiled_templates/8f/8f7402d53ba59939f51da872ef5a32561a8d9fa0abbcfe518474f8c0636388bb.php(63): Twig\Template->display()
#21 /var/www/vendor/twig/twig/src/Template.php(405): __TwigTemplate_856c9a7468d2047aa83872d3500f724f2fb903bbe1205040b0a2ae5219d21474->doDisplay()
#22 /var/www/vendor/twig/twig/src/Template.php(378): Twig\Template->displayWithErrorHandling()
#23 /var/www/vendor/twig/twig/src/Template.php(390): Twig\Template->display()
#24 /var/www/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render()
#25 /var/www/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render()
#26 /var/www/vendor/craftcms/cms/src/web/View.php(392): Twig\Environment->render()
#27 /var/www/vendor/craftcms/cms/src/web/View.php(453): craft\web\View->renderTemplate()
#28 /var/www/vendor/craftcms/cms/src/web/Controller.php(238): craft\web\View->renderPageTemplate()
#29 /var/www/vendor/craftcms/commerce/src/controllers/OrdersController.php(96): craft\web\Controller->renderTemplate()
#30 [internal function]: craft\commerce\controllers\OrdersController->actionOrderIndex()
#31 /var/www/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#32 /var/www/vendor/yiisoft/yii2/base/Controller.php(180): yii\base\InlineAction->runWithParams()
#33 /var/www/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction()
#34 /var/www/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction()
#35 /var/www/vendor/craftcms/cms/src/web/Application.php(274): yii\base\Module->runAction()
#36 /var/www/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction()
#37 /var/www/vendor/craftcms/cms/src/web/Application.php(259): yii\web\Application->handleRequest()
#38 /var/www/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest()
#39 /var/www/web/index.php(21): yii\base\Application->run()
#40 {main}
Additional Information:
Array
(
    [0] => 42S22
    [1] => 1054
    [2] => Unknown column 'commerce_orders.orderSiteId' in 'field list'
)

```

**To Reproduce**
How, and what happened?
1. Install Craft & Commerce
2. Navigate to `/control-panel/commerce/orders`

**Expected behavior**
It should load the Orders. 

**Additional info**
- Craft version: 3.5.14
- Commerce version: 3.2.9
- PHP version: 7.4.11
- Database driver & version: MySQL 5.5.5
- Other Plugins & versions: 

Plugins | Versions
-- | --
Craft Commerce | 3.2.9
OTP SimplePay for Craft Commerce | 1.1.0

**Comment by andris-sevcenko** [comment]
This has just been fixed in Commerce 3.2.9.1!

**Comment by Saboteur777** [comment]
Thanks!
